openapi/publicapi_spec.json

```diff
@@ -1549,6 +1549,12 @@
             "type" : "string",
             "example" : "1234",
             "readOnly" : true
+          },
+          "wallet_type" : {
+            "type" : "string",
+            "description" : "The digital wallet type that the user paid with",
+            "enum" : [ "APPLE_PAY", "GOOGLE_PAY" ],
```

Comment:
Not sure how this `enum` field appears

Reply:
have updated the schema deff in the `CardDetails` class with allowable values, still shows up as an enum though 🤷🏻‍♂️

Reply:
It's correct now, which is what matters. Before it wasn't correct, but I couldn't see how it was being generated at all